bitsail-connectors/connector-hbase/src/main/java/com/bytedance/bitsail/connector/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -0,0 +1,88 @@
+
+    private static final Integer BIT_LENGTH = 1;
+
+    private BitSailConfiguration configuration;
+
+    public HBaseDeserializationFormat(BitSailConfiguration configuration) {
+        this.configuration = configuration;
+    }
+
+    @Override
+    public DeserializationSchema<byte[][], Row> createRuntimeDeserializationSchema(TypeInfo<?>[] typeInfos) {
+        return new DeserializationSchema<byte[][], Row>() {
+
+            @Override
+            public boolean isEndOfStream(Row nextElement) {
+                return false;
+            }
+
+            @Override
+            public Row deserialize(byte[][] rowCell) {
+                Row row = new Row(typeInfos.length);
+                for (int i = 0; i < row.getArity(); i++) {
+                    TypeInfo<?> typeInfo = typeInfos[i];
+                    Column column = deserializeValue(typeInfo, rowCell[i]);
+                    row.setField(i, column);
+                }
+                return row;
+            }
+
+            private Column deserializeValue(TypeInfo<?> typeInfo, byte[] cell) throws BitSailException {
+                Class<?> columnTypeClass = typeInfo.getTypeClass();
+
+                if (TypeInfos.STRING_TYPE_INFO.getTypeClass() == columnTypeClass) {
+                    return new StringColumn(cell == null ? null : new String(cell, Charset.defaultCharset()));
+
+                } else if (TypeInfos.BOOLEAN_TYPE_INFO.getTypeClass() == columnTypeClass) {
+                    if (cell != null && cell.length > BIT_LENGTH) {
+                        return new BooleanColumn(Boolean.valueOf(Bytes.toString(cell)));
+                    }
+                    return new BooleanColumn(cell == null ? null : Bytes.toBoolean(cell));
+
+                } else if (TypeInfos.INT_TYPE_INFO.getTypeClass() == columnTypeClass ||
+                        TypeInfos.SHORT_TYPE_INFO.getTypeClass() == columnTypeClass) {
+                    return new LongColumn(cell == null ? null : Integer.valueOf(Bytes.toString(cell)));
+
+                } else if (columnTypeClass == TypeInfos.LONG_TYPE_INFO.getTypeClass()
+                        || columnTypeClass == TypeInfos.BIG_INTEGER_TYPE_INFO.getTypeClass()) {
+                    return new LongColumn(cell == null ? null : Long.valueOf(Bytes.toString(cell)));
+
+                } else if (columnTypeClass == TypeInfos.DOUBLE_TYPE_INFO.getTypeClass()
+                        || columnTypeClass == TypeInfos.FLOAT_TYPE_INFO.getTypeClass()) {
+                    return new DoubleColumn(cell == null ? null : Double.valueOf(Bytes.toString(cell)));
+
+                } else if (columnTypeClass == TypeInfos.SQL_DATE_TYPE_INFO.getTypeClass() ||
+                        columnTypeClass == TypeInfos.SQL_TIME_TYPE_INFO.getTypeClass() ||
+                        columnTypeClass == TypeInfos.SQL_TIMESTAMP_TYPE_INFO.getTypeClass()) {
+                    return new DateColumn(cell == null ? null : Long.valueOf(Bytes.toString(cell)));
+
```

Comment:
Make the equal pattern consistent? I can see two types of pattern:
1. `TypeInfos.STRING_TYPE_INFO.getTypeClass() == columnTypeClass`
2. `columnTypeClass == TypeInfos.LONG_TYPE_INFO.getTypeClass()`

Personally prefer using switch-case instead of if-else in this case.

Reply:
Made the equal pattern consistent based on suggestion.